kubernetes/terraform/dev/cluster.tf

```diff
@@ -40,7 +40,7 @@ resource "digitalocean_kubernetes_node_pool" "cloud-services-node-pool" {
 
   name       = "cloud-services-node-pool"
   size       = "s-2vcpu-4gb"
-  node_count = 3
+  node_count = 4
```

Comment:
had to do this to scale it in dev, but did it manually since I couldn't run terraform successfully (the older version of tf doesn't run on my mac)